src/planscape/forsys/get_forsys_inputs.py

```diff
@@ -150,12 +165,19 @@ def __init__(
                 area * self.TREATMENT_COST_PER_KM_SQUARED)
 
 
+# This is limited to a single scenario.
 class ForsysGenerationInput():
     # Treatment cost per kilometer-squared (in USD)
     # TODO: make this variable based on a user input and/or a treatment cost
     # raster.
     TREATMENT_COST_PER_KM_SQUARED = 5000 * 1000 * 1000
 
+    # For condition scores normalized to the range, [-1, 1], maximum priority
+    # weight is 2.0. This constant is necessary for normalizing scores when
```

Comment:
we allow users to set max priority to be anywhere between 1 and 5 in the front end. is this something different? if so, can we use a different term than priority to reduce ambiguity.

Reply:
I'll rename this. It's something different.